openedx/core/djangoapps/schedules/models.py

```diff
@@ -54,12 +54,38 @@ class ScheduleConfig(ConfigurationModel):
     KEY_FIELDS = ('site',)
 
     site = models.ForeignKey(Site, on_delete=models.CASCADE)
-    enqueue_recurring_nudge = models.BooleanField(default=False)
-    deliver_recurring_nudge = models.BooleanField(default=False)
-    enqueue_upgrade_reminder = models.BooleanField(default=False)
-    deliver_upgrade_reminder = models.BooleanField(default=False)
-    enqueue_course_update = models.BooleanField(default=False)
-    deliver_course_update = models.BooleanField(default=False)
+    enqueue_recurring_nudge = models.BooleanField(
+        default=False,
+        help_text=_('Enable message queuing for recurring nudge schedule.'),
+    )
+    deliver_recurring_nudge = models.BooleanField(
+        default=False,
+        help_text=_('Enable message delivery for recurring nudge schedule.'),
+    )
+    enqueue_upgrade_reminder = models.BooleanField(
+        default=False,
+        help_text=_('Enable message queuing for upgrade reminder schedule.'),
+    )
+    deliver_upgrade_reminder = models.BooleanField(
+        default=False,
+        help_text=_('Enable message delivery for upgrade reminder schedule.'),
+    )
+    enqueue_course_update = models.BooleanField(
+        default=False,
+        help_text=_('Enable message queuing for course update schedule.'),
+    )
+    deliver_course_update = models.BooleanField(
+        default=False,
+        help_text=_('Enable message delivery for course update schedule.'),
+    )
+    enqueue_course_due_date_reminder = models.BooleanField(
+        default=False,
+        help_text=_('Enable message queuing for due date reminder schedule.'),
+    )
+    deliver_course_due_date_reminder = models.BooleanField(
+        default=False,
+        help_text=_('Enable message delivery for due date reminder schedule.'),
+    )
```

Comment:
What is the difference between message queuing and message delivery? It may be hard to understand when you read it in Django admin without additional context.

Reply:
It is bit weird, queuing enables the process of the given schedule while delivery to enable sending emails once the process it complete.